### DISCUSSION TITLE
How do I ignore a gradle project in my monorepo?

### DISCUSSION THREAD

**Discussion by naipofo** [redhat-developer/vscode-java#3471] [Q&A] [answered]
I'm building a monorepo with both a java mvn project and a flutter front-end app. Flutter also contains an android component made with gradle, so the plugin tries to import it and fails. 
I don't care much about that project, it's automatically generated by flutter and I don't plan on changing anything there, but the java plugin in vscode is *very* adamant on importing it.
I tried changing my workspace settings: `java.import.exclusions`, `java.project.resourceFilters`, `maven.excludedFolders`, `java.import.gradle.enabled` and `java.import.gradle.wrapper.enabled`; 
but none of them prevented the plugin from trying to import the gradle file.
![image](https://github.com/redhat-developer/vscode-java/assets/50967056/6864e734-bed5-4726-9a0c-9fa392182d9e)

Is there a way to disable the importing?
I have the whole java extension pack installed, I'm on 1.83.1

**Comment by testforstephen** [comment, marked as answer]
You can disable gradle importer by the setting `"java.import.gradle.enabled": false`, and then clean workspace via "**F1 -> Java: Clean Java Language Server Workspace**".
